packages/client/src/components/PlayersList/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, {FC} from 'react';
+import styles from './playersList.module.scss';
+import {TPlayersListProps} from './typing';
+import {PlayersListTable} from './Table';
+
+export const PlayersList: FC<TPlayersListProps> = React.memo((props) => {
+
+	const {data, showIndex = false} = props;
+
+	return (
+		<div className={styles.wrapper}>
+			<div data-column='whiteline'></div>
+			<div className={styles.table} data-showindex={showIndex}>
+				<div data-column='index'>Место</div>
+				<div>Игрок</div>
+				<div>Побед онлайн</div>
+				<div>Побед оффлайн</div>
+				<PlayersListTable data={data}/>
```

Comment:
Смотрится уже лаконичней, можно и так оставить. или переделать аля:
```suggestion js
				{data.map((playerData, index) => (<PlayerListItem key={palyerData.id} playerData={playerData} idx={index} />))}				
```

Решать тебе

Reply:
Неплохо, возьму на заметку! Спасибо!